src/Alpha_complex/utilities/alpha_complex_persistence.cpp

```diff
@@ -75,18 +79,26 @@ Simplex_tree create_simplex_tree(const std::string &off_file_points, const std::
     // Init of an alpha complex from an OFF file
     Gudhi::alpha_complex::Alpha_complex<Kernel, true> alpha_complex_from_file(points, weights);
 
-    if (!alpha_complex_from_file.create_complex(stree, alpha_square_max_value, exact_version)) {
-      std::cerr << "Alpha complex simplicial complex creation failed." << std::endl;
-      exit(-1);
-    }
+    if (output_squared_root_values)
+      complex_creation = alpha_complex_from_file.template create_complex<false>(stree, alpha_square_max_value,
+                                                                               exact_version);
+    else
+      complex_creation = alpha_complex_from_file.template create_complex<true>(stree, alpha_square_max_value,
+                                                                                exact_version);
```

Comment:
It is in the right order for `output_squared_root_values`, but if we change it to `output_squared_values`, it needs to be the other way around.

Reply:
Modified by [44a6747](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/44a6747c1c2ffd0d80db14bfd65d64cd9f0e6e88), [4227322](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/4227322a044ffd87c2a0d3743d8bd9a6f1d66bf1) and [404cb39](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/404cb3951e1c9f6283a2ccc3c39ce340b65f9ddd)